x/feegrant/keeper/keeper.go

```diff
@@ -33,13 +34,13 @@ type Keeper struct {
 var _ ante.FeegrantKeeper = &Keeper{}
 
 // NewKeeper creates a feegrant Keeper
-func NewKeeper(cdc codec.BinaryCodec, storeService store.KVStoreService, ak feegrant.AccountKeeper) Keeper {
-	sb := collections.NewSchemaBuilder(storeService)
+func NewKeeper(env appmodule.Environment, cdc codec.BinaryCodec, ak feegrant.AccountKeeper) Keeper {
```

Comment:
again nit keeping the params in the same place

```suggestion
func NewKeeper(cdc codec.BinaryCodec, env appmodule.Environment, ak feegrant.AccountKeeper) Keeper {
```